pyeudiw/tests/settings.py

```diff
@@ -240,7 +240,7 @@
             "config": {
                 "client_id": f"{BASE_URL}/OpenID4VP",
                 "relying_party_certificate_chains_by_ca":{
-                    f"{BASE_URL}/OpenID4VP": gen_chain()
+                    f"{BASE_URL}/OpenID4VP": gen_chain(ca_cn=BASE_URL.replace("https://", "")),
```

Comment:
This MUST be the entity of the CA

the rp is not the CA

RP's client id != CA entity id

Reply:
Please invent an ACME CA not using data belonging to the rp